README.md

```diff
@@ -21,6 +21,7 @@ of them (yet).
 * `Braintree::CreditCard.find`
 * `Braintree::CreditCard.sale`
 * `Braintree::CreditCard.update`
+* `Braintree::CreditCard.delete`
```

Comment:
This is a silly little thing, but I'm trying to keep the method list alphabetized - can you move it to right above `.find`?